yarpcerrors/errors.go

```diff
@@ -57,7 +55,9 @@ func FromError(err error) *Status {
 	}
 }
 
-// IsStatus returns true if err is a Status.
+// IsStatus returns whether the provided error is a YARPC error.
```

Comment:
Why is this changed? You seem to clarify in `FromError` that this relies on this being a `Status`, but then remove it here.

Reply:
Eh, this is fine, let's just leave it.